website/docs/d/multicast_pool_block.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "Access Policies"
+layout: "aci"
+page_title: "ACI: aci_multicast_pool_block"
+sidebar_current: "docs-aci-data-source-multicast_pool_block"
+description: |-
+  Data source for ACI the Multicast Address Pool Block
+---
+
+# aci_multicast_pool_block #
+
+Data source for the ACI the Multicast Address Pool Block
```

Comment:
Data source for the ACI Multicast Address Pool Block

Reply:
ok